### PR TITLE
Document the database schema in skyportal using autodoc and numpydoc

### PR DESCRIPTION
**Pull request by dannygoldstein**
This PR adds:

   1. Documentation for all attributes, methods, columns, etc. of classes that implement `skyportal.models.Base`.
   2. Automated generation of user-facing docs for these classes using sphinx+numpydoc.

### CODE REVIEW

**Comment by dannygoldstein** [comment]
Screen Shot 2020-08-19 at 7 47 30 PM
Screen Shot 2020-08-19 at 7 48 07 PM

**Comment by dannygoldstein** [comment]
[ch1198]

**Comment by dannygoldstein** [comment]
green

File: skyportal/models.py [hunk 1]
```diff
@@ -72,57 +72,99 @@ def is_owned_by(self, user_or_token):
...
+    sa.Boolean,
+    nullable=False,
+    default=False,
+    doc="Boolean flag indicating whether the User " "is an admin of the group.",
+)
 
 
 class Stream(Base):
-    name = sa.Column(sa.String, unique=True, nullable=False)
+    """A stream producing """
```
Comment (acrellin) [inline skyportal/models.py:165]: Sounds like this was intended to be continued

File: skyportal/models.py [hunk 2]
```diff
@@ -72,57 +72,99 @@ def is_owned_by(self, user_or_token):
...
 
 GroupUser = join_model('group_users', Group, User)
-GroupUser.admin = sa.Column(sa.Boolean, nullable=False, default=False)
+GroupUser.__doc__ = "Join table mapping `Group`s to `User`s."
+
+GroupUser.admin = sa.Column(
+    sa.Boolean,
+    nullable=False,
+    default=False,
+    doc="Boolean flag indicating whether the User " "is an admin of the group.",
```
Comment (acrellin) [inline skyportal/models.py:160]:
```suggestion
    doc="Boolean flag indicating whether the User is an admin of the group.",
```

File: skyportal/models.py [hunk 3]
```diff
@@ -72,57 +72,99 @@ def is_owned_by(self, user_or_token):
...
         secondary='group_streams',
         back_populates='groups',
         passive_deletes=True,
+        doc='Stream access required for a User to become a member of the Group.',
+    )
+    filters = relationship(
+        "Filter",
+        back_populates="group",
+        passive_deletes=True,
+        doc='All filters (not just active) associated with ' 'a group.',
```
Comment (acrellin) [inline skyportal/models.py:104]:
```suggestion
        doc='All filters (not just active) associated with a group.',
```

File: skyportal/models.py [hunk 4]
```diff
@@ -72,57 +72,99 @@ def is_owned_by(self, user_or_token):
...
         cascade='save-update, merge, refresh-expire, expunge',
         passive_deletes=True,
+        doc='Elements of a join table mapping Users to Groups.',
     )
 
-    observing_runs = relationship('ObservingRun', back_populates='group')
+    observing_runs = relationship(
+        'ObservingRun',
+        back_populates='group',
+        doc='The observing runs associated with ' 'this group.',
```
Comment (acrellin) [inline skyportal/models.py:126]:
```suggestion
        doc='The observing runs associated with this group.',
```

File: skyportal/models.py (hunk 1 above)
Comment (dannygoldstein) [reply skyportal/models.py]: fixed

**Comment by dannygoldstein** [comment]
@profjsb failure on GA is unrelated

**Pull request merged**